src/Behat/MinkExtension/Context/MinkContext.php

```diff
@@ -392,29 +483,36 @@ public function assertCheckboxChecked($checkbox)
     }
 
     /**
-     * Checks, that checkbox with specified in|name|label|value is unchecked.
+     * Checks, that (?P<num>\d+) CSS elements exist on the page
+     * Example: Then I should see 5 "div" elements
+     * Example: And I should see 5 "div" elements
      *
-     * @Then /^the "(?P<checkbox>(?:[^"]|\\")*)" checkbox should not be checked$/
-     * @Then /^the checkbox "(?P<checkbox>(?:[^"]|\\")*)" should (?:be unchecked|not be checked)$/
-     * @Then /^the checkbox "(?P<checkbox>(?:[^"]|\\")*)" is (?:unchecked|not checked)$/
+     * @Then /^(?:|I )should see (?P<num>\d+) "(?P<element>[^"]*)" elements?$/
      */
-    public function assertCheckboxNotChecked($checkbox)
+    public function assertNumElements($num, $element)
```

Comment:
Actually, he has not added new methods. He changed the order of existing methods.

@sajjadhossain please revert this change. Methods should not be reordered. It makes the review a pain